operator/src/network/ipfs.rs

```diff
@@ -253,14 +263,28 @@ impl RustIpfsConfig {
                 ..Default::default()
             }),
             volume_mounts: Some(vec![VolumeMount {
-                mount_path: "/data/ipfs".to_owned(),
+                mount_path: IPFS_STORE_DIR.to_owned(),
                 name: IPFS_DATA_PV_CLAIM.to_owned(),
                 ..Default::default()
             }]),
             security_context: net_config.debug_mode.then(debug_mode_security_context),
             ..Default::default()
         }
     }
+
+    fn init_container(&self, net_config: &NetworkConfig) -> Option<Container> {
+        self.migration_cmd.as_ref().map(|cmd| Container {
+            name: "ipfs-migration".to_string(),
+            command: Some(
+                vec!["/usr/bin/ceramic-one", "migrations"]
+                    .into_iter()
+                    .chain(cmd.iter().map(String::as_str))
```

Comment:
If we moved the `chain` below the `map` on the next line, could we take out the `as_str` mapping?

Reply:
No, we do not own the strings in `cmd` so the `as_str` converts from `&String` to `&str`. We could instead convert directly to `String` but that still requires a `map` on the `cmd` iterator.

Reply:
So its about the same either way, I don't have a strong reason for one over the other.

Reply:
Ah understood, thanks.